docs/integration_quality_scale_index.md

```diff
@@ -17,51 +17,53 @@ This integration passes the bare minimum requirements to become part of the inde
-- Entities have unique ID (if available) ([docs](entity_registry_index.md#unique-id-requirements))
+- Configuration:
+  - Configuration is done using config entries and data is stored in `config_entry.runtime_data`.
+  - Raise [`ConfigEntryNotReady`](integration_setup_failures.md) if unable to connect during platform setup (if appropriate)
+- Devices and entities:
+  - Set an appropriate `SCAN_INTERVAL` (if a polling integration)
+  - Handles internet unavailable. Log a warning once when unavailable, log once when reconnected.
+  - Handles device/service unavailable. Log a warning once when unavailable, log once when reconnected.
+  - Set `available` property to `False` if appropriate ([docs](core/entity.md#generic-properties))
+  - Entities have unique ID (if available) ([docs](entity_registry_index.md#unique-id-requirements))
+- Services:
+  - Operations like service calls and entity methods (e.g. *Set HVAC Mode*) have proper exception handling. Raise `ServiceValidationError` on invalid user input and raise `HomeAssistantError` for other failures such as a problem communicating with a device. [Read more](/docs/core/platform/raising_exceptions) about raising exceptions.
+- Tests:
+  - Full test coverage for the config flow
 
 ## Gold 🥇
 
-This is a solid integration that is able to survive poor conditions and can be configured via the user interface.
+This is a solid integration that offers the best possible user experience that Home Assistant has to offer.
 
 - Satisfying all Silver level requirements.
-- Configurable via config entries.
-  - Don't allow configuring already configured device/service (example: no 2 entries for same hub)
-  - Discoverable (if available)
-  - Set unique ID in config flow (if available)
-  - Raise [`ConfigEntryNotReady`](integration_setup_failures.md#integrations-using-async_setup_entry) if unable to connect during entry setup (if appropriate)
-- Entities have device info (if available) ([docs](device_registry_index.md#defining-devices))
+- Configuration:
+  - Don't allow configuring already configured device/service (example: no 2 entries for same hub). If possible, set a unique ID on the config flow.
+  - [Discoverable](config_entries_config_flow_handler.md#discovery-steps) (if available)
+  - Support config entry [unloading](config_entries_index.md#unloading-entries) and [removal](config_entries_index.md#removal-of-entries)
+  - Handles expiration of auth credentials. Refresh if possible or print correct error and fail setup. If based on a config entry, should trigger a new config entry flow to re-authorize.  ([docs](integration_setup_failures.md))
+- Devices and entities:
+  - Entities have [device info](device_registry_index.md#defining-devices) and uses [`has_entity_name = True`](./core/entity.md#has_entity_name-true-mandatory-for-new-integrations)
+  - Entities have correct device classes where appropriate ([docs](core/entity.md#generic-properties))
```

Comment:
I think this should be bare requirement of silver, as this involves translations and naming as well.